hudi-utilities/src/main/java/org/apache/hudi/utilities/sources/AvroKafkaSource.java

```diff
@@ -52,6 +54,10 @@
   // these are native kafka's config. do not change the config names.
   private static final String NATIVE_KAFKA_KEY_DESERIALIZER_PROP = "key.deserializer";
   private static final String NATIVE_KAFKA_VALUE_DESERIALIZER_PROP = "value.deserializer";
+  // These are settings used to pass things to KafkaAvroDeserializer
+  public static final String KAFKA_AVRO_VALUE_DESERIALIZER_SCHEMA = "hoodie.deltastreamer.source.kafka.value.deserializer.schema";
+  public static final String KAFKA_AVRO_VALUE_DESERIALIZER_PROPERTY_PREFIX = "hoodie.deltastreamer.source.kafka.value.deserializer.";
```

Comment:
can we move this ahead of line 58 and use this to form  the other variables. 
for eg:
```
public static final String KAFKA_AVRO_VALUE_DESERIALIZER_SCHEMA = KAFKA_AVRO_VALUE_DESERIALIZER_PROPERTY_PREFIX + "schema";
```

Reply:
Thanks for the suggestion, fixed